src/fastcs/transport/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .epics.options import EpicsDocsOptions as EpicsDocsOptions
+from .epics.options import EpicsGUIOptions as EpicsGUIOptions
+from .epics.options import EpicsIOCOptions as EpicsIOCOptions
+from .epics.options import EpicsOptions as EpicsOptions
+from .tango.options import TangoDSROptions as TangoDSROptions
+from .tango.options import TangoOptions as TangoOptions
+
+__all__ = ["EpicsOptions", "TangoOptions"]
```

Comment:
We don't need the `__all__` here because of the import as

Reply:
Agreed. I also noticed I am missing "RestOptions". Messed up the rebase here